internal/integration/rest/models.go

```diff
@@ -63,6 +63,26 @@ type (
 		ParendID int    `json:"parentId"`
 	}
 
+	Project struct {
+		ID                 int                   `json:"id"`
+		TeamID             int                   `json:"teamId"`
+		Name               string                `json:"name"`
+		IsPublic           bool                  `json:"isPublic"`
+		SourceSettingsLink SourceSettingsLinkObj `json:"sourceSettingsLink"`
+		Link               LinkObj               `json:"link"`
```

Comment:
The link attribute should be for API navigation only, and not needed for the export.